### ISSUE TITLE
Linux

### ISSUE THREAD

**Issue by Sciss** [JosephRedfern/VarienseVMU#2]
Sorry to 'abuse' the issue tracker for this. I would like to enquire if you are using the VMC931 under Linux? I read there were some initial problems but that they have been fixed; I'm eying the VMC, so I want to be sure I see people using it successfully under Linux. Thanks

**Comment by JosephRedfern** [comment]
Hi @Sciss. I no longer have access to a VMU931 device, so it's hard for me to test for certain.

There certainly used to be issues communicating with the VMU931 on at least some Linux machines -- I ended up having to stream VMU931 packets over the network from my Mac to my Linux desktop as a work around.

That said, I believe that Linux compatibility issues have been fixed (going by: https://variense.com/blog/patch-note-vmu931-v1-0-1/). I know of some people that have successfully used PyVMU on the Raspberry Pi (under Linux), but unfortunately I can't check my original setup without acquiring another VMU931.
